e2e/nomostest/nt.go

```diff
@@ -919,8 +920,14 @@ func (nt *NT) autopilotClusterSupportsBursting() (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	versions := strings.Split(strings.TrimSpace(string(out)), " ")
-	for _, nodeVersion := range versions {
+	outStr := strings.TrimSpace(string(out))
+	if outStr == "" {
+		// No nodes (scaled to zero)
```

Comment:
If there aren't any nodes, can we be certain that the cluster is running on the minimum version?

Reply:
If it weren't, the cluster version check would have failed, right?

Reply:
Oh right, we only checks the major version. I guess we need to check the minor too?

Reply:
Wait. Nah. We check the major and minor of the cluster.

Reply:
We check the initial cluster version above for the first requirement (cluster was created with version 1.26 or later).

This check is to ensure the second requirement (current cluster version is `1.30.2-gke.1394000` or later). If we skip this check I don't think we can be certain that the current cluster version meets the second requirement.

Reply:
Turns out the gcloud response has all the info we need. So I changed it to just parse the json response and check the 3 version fields available there.

I don't know if the `currentNodeVersion` changes before or after all the nodes have been updated, but it probably shouldn't matter in practice. If it fails in CI we can be more defensive and add the check back in to loop through all the nodes or maybe all the node pools in the json response.